### PR TITLE
Skip failing torch post commit test

### PR DESCRIPTION
**Pull request by AnandInguva**
gs://apache-beam-ml/testing/inputs/it_coco_validation_inputs.txt was changed on July 21st and it needs to be addressed once the team gets back

https://github.com/apache/beam/issues/22413

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by AnandInguva** [comment]
R: @TheNeuralBit

**Comment by AnandInguva** [comment]
Run Python 3.7 PostCommit

**Comment by AnandInguva** [comment]
Run Python 3.9 PostCommit

File: sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def test_torch_run_inference_imagenet_mobilenetv2(self):
 
   @pytest.mark.uses_pytorch
   @pytest.mark.it_postcommit
+  @pytest.mark.skip
```
Comment (TheNeuralBit) [inline sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py:96]:
```suggestion
  @pytest.mark.skip # TODO(https://github.com/apache/beam/issues/22413): Debug input data issue and unskip
```

This should link to the relevant issue.

**Review by TheNeuralBit — APPROVED** [review]

**Comment by TheNeuralBit** [comment]
PostCommit runs:
- https://ci-beam.apache.org/job/beam_PostCommit_Python37_PR/374/
- https://ci-beam.apache.org/job/beam_PostCommit_Python39_PR/32/

File: sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ def test_torch_run_inference_imagenet_mobilenetv2(self):
 
   @pytest.mark.uses_pytorch
   @pytest.mark.it_postcommit
+  @pytest.mark.skip # TODO(https://github.com/apache/beam/issues/22413): Debug input data issue and unskip
```
Comment (TheNeuralBit) [inline sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py:96]:
```suggestion
  @pytest.mark.skip  # TODO(https://github.com/apache/beam/issues/22413): Debug input data issue and unskip
```

**Comment by TheNeuralBit** [comment]
Python_PVR_Flink failure is unrleated, and PostCommits passed.

**Comment by AnandInguva** [comment]
Thanks @TheNeuralBit

**Pull request merged**